packages/mint-components/src/components/sqm-image/sqm-image-view.tsx

```diff
@@ -0,0 +1,51 @@
+import { h, VNode } from "@stencil/core";
+import { createStyleSheet } from "../../styling/JSS";
+
+export interface ImageViewProps {
+  imageUrl: string;
+  left?: string;
+  right?: string;
+  alignment?: "left" | "center" | "right";
+  backgroundColor?: string;
+  minHeight?: string; // helps with constraining minimum size
+  maxWidth?: string; // helps with constraining maximum size
```

Comment:
This prop was deleted, should be removed from code

Reply:
removed `maxWidth`!